src/EditorFeatures/Core/EditorFeaturesResources.Designer.cs

```diff
@@ -1918,36 +1936,32 @@ internal class EditorFeaturesResources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Toggling block comment....
+        ///   Looks up a localized string similar to Toggle Line Comment.
```

Comment:
>  ///   Looks up a localized string similar to Toggle Line Comment. [](start = 6, length = 67)

I hope this is an automated change not an intended one for the PR.

Reply:
I didn't change this, should be automated